Client/Frontend/Login Management/LoginListViewModelHelper.swift

```diff
@@ -50,15 +62,7 @@ class LoginListDataSourceHelper {
         var titleSet = Set<Character>()
 
         return deferDispatchAsync(DispatchQueue.global(qos: DispatchQoS.userInteractive.qosClass)) {
-            // Precompute the baseDomain, host, and hostname values for sorting later on. At the moment
-            // baseDomain() is a costly call because of the ETLD lookup tables.
-            logins.forEach { login in
-                self.domainLookup[login.id] = (
-                    login.hostname.asURL?.baseDomain,
-                    login.hostname.asURL?.host,
-                    login.hostname
-                )
-            }
+            self.setDomainLookup(logins)
```

Comment:
This should probably be a weak self to reduce possibility of a retain cycle, since it's in an asynchronous closure